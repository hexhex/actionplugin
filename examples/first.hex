#first_action[]{c_p,1}[:].
