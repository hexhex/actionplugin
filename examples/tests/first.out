aux_a_10000000_0
original: #first_action[]{b,1}[,]
rewritten: aux_a_10000000_0(first_action,b,1,1,1).
{}